modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/core/components/PageRenderer/EditorVariant.es.js

```diff
@@ -167,7 +167,7 @@ export const Column = ({
 						ref={(node) => {
 							if (
 								allowNestedFields &&
-								!rootParentField.ddmStructureId
```

Comment:
Hey @marcosapmf just a consideration about this change, maybe it doesn’t fix 100%. ParentField only checks the nearest field, in nested objects it will not be able to see a FieldSet.

The most appropriate solution for this is to modify the Layout, it is rendered nested. The Field of FieldSet calls the Layout to render the nested fields, currently, it deactivates the `editable` but the Layout does not respect this, that is, it reuses the mixins of components initially rendered in the structure, in this case we should just revalidate the mixins and reassemble again respected the property of `editable`. This will allow the Layout inside the FieldSet not to call the EditorVariant, so we can remove all these checks from the EditorVariant side.

If you wish I can send a follow-up PR to fixes this.